warehouse/oidc/models/activestate.py

```diff
@@ -0,0 +1,202 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import urllib
+
+from typing import Any
+
+from sqlalchemy import ForeignKey, String, UniqueConstraint
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import Query, mapped_column
+
+import warehouse.oidc.models._core as oidccore
+
+from warehouse.oidc.errors import InvalidPublisherError
+from warehouse.oidc.interfaces import SignedClaims
+from warehouse.oidc.models._core import (
+    CheckClaimCallable,
+    OIDCPublisher,
+    PendingOIDCPublisher,
+)
+
+ACTIVESTATE_URL = "https://platform.activestate.com"
+
+ACTIVESTATE_OIDC_API_PATH = "api/v1/oauth/oidc"
```

Comment:
Nitpick: These look slightly duplicated against `utils.ACTIVESTATE_OIDC_ISSUER_URL`

Reply:
(`ACTIVESTATE_OIDC_API_PATH` seems unused, so maybe it can be removed outright? And `ACTIVESTATE_URL` is used only within this module, so maybe it could be made into `_ACTIVESTATE_BASE_URL` or similar to emphasize that it's private.)

Reply:
Good eye.  Changes incoming.